src/components/Countries/Countries.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import Header from '../Header/Header';
+import Footer from '../Footer/Footer';
+import './countries.css';
+
+const Countries = () => {
+  const countries = useSelector((state) => state.reduceData);
+
+  const regionCountry = countries.filter((country) => country.continent === 'Europe');
+  const container = regionCountry.map((country) => (
+    <Link to={`/Details/${country.name}`} key={country.name} className="container_country">
+      <img src={country.flag} alt={country.name} width={100} className="country_flag" />
+      <div className="cases">
+        <h3 className="country_name">
+          {country.name}
+        </h3>
+        <p className="case">
+          {country.cases}
+        </p>
+      </div>
+
+    </Link>
+  ));
+  const handleChange = () => {
+    let text = '';
+    const input = document.querySelector('#input');
+    const inputCountry = input.value.toUpperCase();
+    const containerCountry = document.querySelectorAll('.container_country');
+    [...containerCountry].forEach((country) => {
+      const cont = country;
+      text = cont.innerText || cont.textContent;
+      if (text.toUpperCase().indexOf(inputCountry) > -1) {
+        cont.style.display = '';
+      } else {
+        cont.style.display = 'none';
+      }
+    });
+  };
```

Comment:
- [ ] From the requirements of this project, we are asked to use a `Filter` stateless component to filter data retrieved from the API.